piv/key_test.go

```diff
@@ -78,6 +78,80 @@ func TestYubiKeySignECDSA(t *testing.T) {
 	}
 }
 
+func TestYubiKeyECDSAKeyAgreement(t *testing.T) {
+	yk, close := newTestYubiKey(t)
+	defer close()
+
+	if err := yk.Reset(); err != nil {
```

Comment:
nit: I don't think this test needs to reset the key (that takes a while). Calling GenerateKey() should overwrite any existing content.

Reply:
Sweet. I was stumbling in the dark there.